app/bundles/LeadBundle/Resources/views/Lead/lead.html.twig

```diff
@@ -550,4 +553,14 @@
     <!--/ right section -->
 </div>
 <!--/ end: box layout -->
+<script>
+    Mautic.removeTagFromLead = function (el, leadId, tagId) {
+    console.log('Tag removed from lead');
+    mQuery(el).removeClass('fa-times').addClass('fa-spinner fa-spin');
+
+    Mautic.ajaxActionRequest('lead:removeTagFromLead', {'leadId': leadId, 'tagId': tagId}, function() {
+        mQuery('#tagLabel' + tagId).fadeOut(300, function() { mQuery(this).remove(); });
+    });
+};
+</script>
```

Comment:
Why is it duplicated here once more? It's already in lead.js, isn't it?

Reply:
@escopecz yes, it is
but for some reason this function is not recognized and throws error in console, I wrote it in description:
`There is a little problem: the function which deletes tag is in lead.js, but console says that there is no such function, I added this function at the end of twig template to show that it works and as I understand how to solve this I'll delete this script from template`

Reply:
You either have to enable the dev mode in `.env.local` file or rebuild the production assets with `bin/console mautic:assets:generate` command. Duplicating isn't the way to go.